tool/site/client-react/src/pages/releasePage.tsx

```diff
@@ -0,0 +1,167 @@
+          />
+          <ButtonList>
+            <ReloadButton/>
+          </ButtonList>
+        </>
+      )}
+    </LongRunningTask>
+  )
+}
+
+const ReleaseForm: React.FC<ReleaseFormProps> = ({
+  onSubmit
+}) => {
+  const initialValues: ReleaseFormValues = {releaseType: null, releasePharmacy: null}
+  return (
+    <Formik<ReleaseFormValues> initialValues={initialValues} onSubmit={values => onSubmit(values)}>
+      {formik =>
+        <Form onSubmit={formik.handleSubmit} onReset={formik.handleReset}>
+          <Fieldset>
+            {/* <RadioField
+              name="releaseType"
+              label="Choose how you want to release prescription(s)"
+              fieldRadios={[
+                {
+                  value: "all",
+                  text: "All nominated prescriptions for the below pharmacy"
+                },
+                {
+                  value: "prescriptionId",
+                  text: "A single prescription by ID"
+                },{
+                  value: "custom",
+                  text: "With a FHIR release message"
+                }
+              ]}
+            /> */}
+            <RadioField
+              name="releasePharmacy"
+              label="Pharmacy to release prescriptions to"
+              fieldRadios={[
+                {
+                  value: "VNFKT",
+                  text: "VNFKT - FIVE STAR HOMECARE LEEDS LTD"
+                },
+                {
+                  value: "YGM1E",
+                  text: "YGM1E - MBBM HEALTHCARE TECHNOLOGIES LTD"
+                },
+                // {
+                //   value: "custom",
+                //   text: "Other"
+                // }
+              ]}
+            />
+          </Fieldset>
+          <ButtonList>
+            <Button type="submit">Release</Button>
+            <BackButton/>
+          </ButtonList>
+        </Form>
+      }
+    </Formik>
+  )
+}
+
+async function sendRelease(
+  baseUrl: string,
+  releaseFormValues: ReleaseFormValues
+): Promise<ReleaseResult> {
+  const release = createRelease(releaseFormValues)
+
+  const response = await axios.post<ReleaseResult>(`${baseUrl}dispense/release`, release)
+  console.log(release)
+  console.log(response)
+
+  return response.data
+}
+
+function createRelease(
+  releaseFormValues: ReleaseFormValues
+): fhir.Parameters {
+  const release: fhir.Parameters = {
+    resourceType: "Parameters",
+    id: uuid.v4(),
+    parameter: [
+      {
+        name: "owner",
+        valueIdentifier: {
+          system: "https://fhir.nhs.uk/Id/ods-organization-code",
+          value: releaseFormValues.releasePharmacy
+        }
+      },
+      {
+        name: "status",
+        valueCode: "accepted"
+      }
+    ]
+  }
+  return release
+}
+
+export default ReleasePage
+
+export interface ReleaseFormValues {
+  releaseType: string,
+  releasePharmacy: string
+}
+
+// todo: genericise? wrap messageExpanders to deconstruct result properties??
```

Comment:
I've made it generic in the errors PR

Reply:
Should we get that merged in?

Reply:
Ah yeah I thought I'd seen that somewhere, yeah that would be good